### ISSUE TITLE
Search should be navigatable by page up/down keys

### ISSUE THREAD

**Issue by weeman1337** [vector-im/element-web#21256]
#### What would you like to do?

An a user I want to be able to navigate through the search results by using the page up/down keys  
so that I am able to reach my search results quickly.

#### Why would you like to do it?

↑

#### How would you like to achieve it?

Page up/down keys should scroll the search result list by ? half/full pages.

### Have you considered any alternatives?

Holding down the up/down arrow keys. But this is slow.

**Comment by aaronraimist** [comment]
Are you talking about searching the contents of a room? You should already be able to scroll with page up and down. If it doesn't work for you then we probably need information about what browser you are using.

If you mean arrow up and arrow down for individual results that would be https://github.com/vector-im/element-web/issues/16788

**Comment by weeman1337** [comment]
Ah no - it is about the spotlight search. Wanted to make that clear by using a label. But I was unable to set it.

**Comment by psavan1655** [comment]
I would like to work on it. First want to clear some doubts here.
We are talking about the search bar, where we search for the rooms and all. Right?

Suggestion: Rather than doing Full / Half page scroll, we can do a fix number, Let's say scroll 10 search results on Page Up/Down. @weeman1337

**Comment by weeman1337** [comment]
> We are talking about the search bar, where we search for the rooms and all. Right?

Nope. It is the new search if you click into the search field on the top left:

![image](https://user-images.githubusercontent.com/6216686/220037470-11c8c812-68e0-4fd5-a667-f6db29c452d2.png)

**Comment by NSV1991** [comment]
If this change is done, won't it override the current functionality implemented in https://github.com/vector-im/element-web/issues/16788 ?

**Comment by t3chguy** [comment]
@NSV1991 that issue is about timeline search, not spotlight search

**Comment by NSV1991** [comment]
@t3chguy Pardon me if my question is confusing. I am trying to understand what are we trying to achieve here.
Below is my implementation locally (not committed yet). Please check the below video and guide me if I am in the right direction

Currently working in spotlight
[Spotlight current working scroll.webm](https://user-images.githubusercontent.com/37620010/230341807-b64b8aef-27be-4f48-a8f7-c4268f18ad98.webm)

Updated new scroll in spotlight using keyboard
[NewUpdatedScroll.webm](https://user-images.githubusercontent.com/37620010/230341930-f20ca09c-190a-4ca7-80d4-0fca32472933.webm)

**Comment by t3chguy** [comment]
@NSV1991 this is asking for the <kbd>Page Up</kbd> and <kbd>Page Down</kbd> keys to work in the spotlight, you seem to have just changed how the scrolling for up/down arrow keys works.

**Comment by NSV1991** [comment]
@t3chguy Does this meet expectation?
https://user-images.githubusercontent.com/37620010/230625165-5b01c20d-a716-43ba-9a0d-a6fb07cac56b.mp4

**Comment by t3chguy** [comment]
@NSV1991 almost, it seems to only be moving about 60% of a page rather than a page though
